File: app-kotlin/src/main/java/com/TritiumGaming/phasmophobiaevidencepicker/firebase/firestore/transactions/store/merchandise/FirestoreMerchandise.java
package com.TritiumGaming.phasmophobiaevidencepicker.firebase.firestore.transactions.store.merchandise;

import androidx.annotation.NonNull;

import com.TritiumGaming.phasmophobiaevidencepicker.firebase.firestore.transactions.store.FirestoreMarketplace;
import com.google.firebase.firestore.CollectionReference;
import com.google.firebase.firestore.DocumentReference;

public class FirestoreMerchandise {

    private static final String
            DOCUMENT_MERCHANDISE = "Merchandise";

    @NonNull
    public static DocumentReference getMerchandiseDocument() throws Exception {
        return FirestoreMarketplace.getStoreCollection().document(DOCUMENT_MERCHANDISE);
    }

}

File: app-view/src/main/java/com/tritiumgaming/phasmophobiaevidencepicker/firebase/firestore/transactions/store/merchandise/FirestoreMerchandise.kt
package com.tritiumgaming.phasmophobiaevidencepicker.firebase.firestore.transactions.store.merchandise

import com.tritiumgaming.phasmophobiaevidencepicker.firebase.firestore.transactions.store.FirestoreMarketplace.Companion.storeCollection
import com.google.firebase.firestore.DocumentReference

class FirestoreMerchandise {

    companion object {
        private const val DOCUMENT_MERCHANDISE = "Merchandise"

        val merchandiseDocument: DocumentReference
            get() = storeCollection.document(DOCUMENT_MERCHANDISE)
    }
}
